stubs/redis/redis/client.pyi

```diff
@@ -60,6 +62,8 @@ _Key = Union[Text, bytes]
 # Lib returns str or bytes depending on Python version and value of decode_responses
 _StrType = TypeVar("_StrType", bound=Union[Text, bytes])
 
+_LockType = TypeVar("_LockType")
```

Comment:
Ideally, we'd use a protocol here to constrain `_LockType`, but this is fine for now.